pkg/controller/iam/iamclient/iamclient_integration_test.go

```diff
@@ -117,7 +140,11 @@ func TestAllGetSetDeletePolicyWithExternalRef(t *testing.T) {
 	}
 	testName := getCurrentFuncName()
 	shouldRun := func(fixture resourcefixture.ResourceFixture, mgr manager.Manager) bool {
-		return shouldRunBasedOnRunAndSkipRegexes(testName, fixture) && fixture.Type == resourcefixture.Basic && testiam.ShouldRunWithExternalRef(fixture) && testiam.FixtureSupportsIAMPolicy(t, smLoader, serviceMetadataLoader, dclSchemaLoader, fixture)
+		return shouldRun(testName, fixture,
```

Comment:
Would this be clearer if we have this:

```
if mustRun(test name, fixture) { return true }
...
```

Reply:
Because the new flag is for "must-only-run" scenario, so refactored it a bit differently.